src/tests/ftest/pool/bad_evict.py

```diff
@@ -63,7 +63,10 @@ def test_evict(self):
             saveduuid = (ctypes.c_ubyte * 16)(0)
             for index, _ in enumerate(saveduuid):
                 saveduuid[index] = self.pool.pool.uuid[index]
-            self.pool.pool.uuid[4] = 244
+            if self.pool.pool.uuid[4] != 244:
+                self.pool.pool.uuid[4] = 244
+            else:
+                self.pool.pool.uuid[4] = 255
```

Comment:
Why 244 and 255?

Reply:
Original bad_evict testcase choose 244 (0xf4) and over-written to byte-4 of pool uuid as a negative testcase, when a pool origin uuid byte-4 is 0xf4 test passed on this negative test.

Reply:
But where does 0xf4 come from? I.e. where is it set, and why THAT number? Looking at 244 and 255 here out of context, they seem like magic numbers

Reply:
This is an old testcase to evict an invalid/junk pool-uuid. We can choose a random number, but it would not gain any coverage.